src/ast/element-descriptor.ts

```diff
@@ -29,3 +43,33 @@ export interface ElementDescriptor extends Descriptor {
 
   abstract?: boolean;
 }
+
+/**
+ * The metadata for a single polymer element
+ */
+export class ElementDescriptor implements Descriptor {
```

Comment:
I agree, but I'd rather stay as-is for a very short time, as I think there's a big refactor coming here next.

I want to move Polymer-specific things into the `polymer/` folder and pull out a plain element descriptor interface to live here. I also want to limit the number of references that the descriptors have into the AST... it shouldn't normally be necessary.